storage_controller/src/service/safekeeper_service.rs

```diff
@@ -323,6 +323,42 @@ impl Service {
         })
     }
 
+    pub(crate) async fn tenant_timeline_create_safekeepers_until_success(
+        self: &Arc<Self>,
+        tenant_id: TenantId,
+        timeline_info: TimelineInfo,
+    ) -> anyhow::Result<()> {
+        const BACKOFF: Duration = Duration::from_secs(5);
+
+        loop {
+            if self.cancel.is_cancelled() {
+                anyhow::bail!("Shut down requested while finalizing import");
```

Comment:
Should we bail here? We are trying to avoid having stuff show up as error message, no?

Reply:
Not sure what you mean. We do bail.

Reply:
I mostly mean from a "is this logged as something bad" point of view. shrug, should be uncommon to hit this.